sylt-compiler/src/typechecker.rs

```diff
@@ -1529,6 +1535,17 @@ impl TypeChecker {
                         vars
                     ),
                 },
+
+                Constraint::Variable => match self.find_type(a) {
+                    Type::Void => err_type_error!(
+                        self,
+                        span,
+                        TypeError::Exotic,
+                        "The `void` has no values and cannot be put in a variable"
```

Comment:
```suggestion
                        "The 'void' type has no values and cannot be put in a variable"
```
I don't remember any backticks in error messages...

Reply:
Habits die hard - but it should probably be " actually...

Reply:
Actually - this is concistent with the other error messages, see lines 658, 647. Not gonna change.